integr8ly/tutorial-web-app
# INTLY-468: Add About Modal with Web App release info

## Motivation
INTLY-468

## What
Added PF4 about modal to display web app release information.

## Why
Feature request to display the web app release info within the web app.

## Verification Steps

1. Click the User menu and select About.
2. Verify that the About modal displays and that the web app version information is displayed.
3. Verify that you can close the About modal by clicking the 'x' close button in the upper right corner.

## Checklist:


## Progress


Added Web app version, user name, browser version, browser OS, and copyright year programmatically to the Integreatly About modal. 

There are some differences that still exist between this and mockup:
- Server name and User Role are placeholders for now, have not figured out how to retrieve that info. Need to verify if they are a requirement.
- Missing pattern that is located about 75% to the right of modal. 

## Additional Notes

About modal screen cap:

![integreatly about modal](https://user-images.githubusercontent.com/39063664/52311413-dcde1d80-2974-11e9-9d52-d9a4c9f4232d.png)
